### DISCUSSION TITLE
New collection role for catalogs

### DISCUSSION THREAD

**Discussion by HadrienGardeur** [opds-community/drafts#50] [Ideas]
A lot of the issues currently opened for [authentication](https://github.com/opds-community/drafts/issues?q=is%3Aissue+is%3Aopen+label%3Atopic%3Aauthentication) are IMO more related to how we describe a catalog than authentication or a user's profile.

Several organizations using OPDS are building what is usually called an "OPDS Catalogs registry" and thanks to OPDS 2.0, it's possible to create a collection role specifically for that.

Should we start a draft dedicated to the `catalogs` role and the metadata that would usually be relevant for such use cases?

Aside from defining a new collection role and metadata, this document could also define how geo-location could work as well (URI template + rel value).

**Comment by llemeurfr** [comment]
This OPDS `catalogs` is now defined in https://github.com/opds-community/drafts/pull/38#issuecomment-873127875  and will soon be proposed as a PR to the spec by @leonardr .
